com.chartboost.mediation/Editor/HeliumIntegrationChecker.cs

```diff
@@ -187,27 +187,28 @@ public static void CheckUnityAdsIntegration()
             {
                 if (!Version.TryParse(unityAdsPackage.version, out var unityAdsVersion))
                     return;
-                if (!unityAdsVersion.Equals(HeliumUnityAdsSupportedVersion))
+                if (!unityAdsVersion.Equals(UnityAdsSupportedVersion))
                 {
                     EditorUtility.DisplayDialog(
-                        HeliumWindowTitle,
-                        $"UnityAds integrated through Unity Package Manager with version: {unityAdsPackage.version}. Helium recommended version is {HeliumUnityAdsSupportedVersion}.\n\nUnexpected behaviors can occur.",
+                        WindowTitle,
+                        $"UnityAds integrated through Unity Package Manager with version: {unityAdsPackage.version}. Chartboost Mediation recommended version is {UnityAdsSupportedVersion}.\n\nUnexpected behaviors can occur.",
                         "Ok");
                 }
                 return;
             }
 
             UncommentUnityAdsDependency();
-            Log("[Helium Integration Checker] UnityAds Ad Adapter Check Completed!", "green");
+            AssetDatabase.Refresh();
+            Log("[Chartboost Mediation Integration Checker] UnityAds Ad Adapter Check Completed!", "green");
         }
 
         /// <summary>
-        /// Used to update all existing adapters by Devs choice. This will utilize current's Helium Package version to override all adapters with such version.
+        /// Used to update all existing adapters by Devs choice. This will utilize current's Chartboost Mediation Package version to override all adapters with such version.
         /// </summary>
-        [MenuItem("Helium/Integration/Force Reimport Adapters", false, 2)]
+        [MenuItem("Chartboost Mediation/Integration/Force Reimport Adapters", false, 2)]
         public static void ForceReimportExistingAdapters()
         {
-            var confirmUpdate = EditorUtility.DisplayDialog(HeliumWindowTitle,
+            var confirmUpdate = EditorUtility.DisplayDialog(WindowTitle,
```

Comment:
Can this name collide with any files from 3rd-parties?? Would `ChartboostMediationWindowTitle` help??